File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/Python/PRAK404-2310817310002-JovanGilbertNatamasindah.py
while 1 :
    print("\nPilih Program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    PILIHAN = int(input("Masukkan Pilihan : "))

    if (PILIHAN == 5) :
        print("Terimakasih, telah menggunakan kalkulator Jovan Gilbert Natamasindah\n")
        break


    if(PILIHAN == 1) :
        NILAIPERTAMA = float(input("Masukkan Nilai Pertama : "))
        NILAIKEDUA = float(input("Masukkan Nilai Kedua : "))
        HASIL = NILAIPERTAMA + NILAIKEDUA
        print(f"Hasil Penjumlahan antara {NILAIPERTAMA:.2f} dengan {NILAIKEDUA:.2f} adalah {HASIL:.2f}\n")     
        
    elif(PILIHAN == 2) :
        NILAIPERTAMA = float(input("Masukkan Nilai Pertama : "))
        NILAIKEDUA = float(input("Masukkan Nilai Kedua : "))
        HASIL = NILAIPERTAMA - NILAIKEDUA
        print(f"Hasil Pengurangan antara {NILAIPERTAMA:.2f} dengan {NILAIKEDUA:.2f} adalah {HASIL:.2f}\n")        
        
    elif(PILIHAN == 3) :
        NILAIPERTAMA = float(input("Masukkan Nilai Pertama : "))
        NILAIKEDUA = float(input("Masukkan Nilai Kedua : "))
        HASIL = NILAIPERTAMA * NILAIKEDUA
        print(f"Hasil Perkalian antara {NILAIPERTAMA:.2f} dengan {NILAIKEDUA:.2f} adalah {HASIL:.2f}\n")          
        
    elif(PILIHAN == 4) :
        NILAIPERTAMA = float(input("Masukkan Nilai Pertama : "))
        NILAIKEDUA = float(input("Masukkan Nilai Kedua : "))
        HASIL = NILAIPERTAMA / NILAIKEDUA
        print(f"Hasil Pembagian antara {NILAIPERTAMA:.2f} dengan {NILAIKEDUA:.2f} adalah {HASIL:.2f}\n")     
        
    else :
        print("Input anda salah, silahkan coba lagi\n")

        

File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/C/PRAK404-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int  PILIHAN;
    float HASIL, NILAIPERTAMA, NILAIKEDUA ;

    while (1) {
        
    printf("\nPilih Program \n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan : ");
    scanf("%d", &PILIHAN);

        if (PILIHAN == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Jovan Gilbert Natamasindah\n\n");
            break;
        }

        if(PILIHAN == 1) {
            printf("Masukkan Nilai Pertama : ");
            scanf("%f", &NILAIPERTAMA);
            printf("Masukkan Nilai Kedua : ");
            scanf("%f", &NILAIKEDUA);
            printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", NILAIPERTAMA, NILAIKEDUA, HASIL = NILAIPERTAMA + NILAIKEDUA);     
        }
        else if(PILIHAN == 2) {
            printf("Masukkan Nilai Pertama : ");
            scanf("%f", &NILAIPERTAMA);
            printf("Masukkan Nilai Kedua : ");
            scanf("%f", &NILAIKEDUA);
            printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", NILAIPERTAMA, NILAIKEDUA, HASIL = NILAIPERTAMA - NILAIKEDUA);     
        }
        else if(PILIHAN == 3) {
            printf("Masukkan Nilai Pertama : ");
            scanf("%f", &NILAIPERTAMA);
            printf("Masukkan Nilai Kedua : ");
            scanf("%f", &NILAIKEDUA);
            printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", NILAIPERTAMA, NILAIKEDUA, HASIL = NILAIPERTAMA * NILAIKEDUA);     
        }
        else if(PILIHAN == 4) {
            printf("Masukkan Nilai Pertama : ");
            scanf("%f", &NILAIPERTAMA);
            printf("Masukkan Nilai Kedua : ");
            scanf("%f", &NILAIKEDUA);
            printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", NILAIPERTAMA, NILAIKEDUA, HASIL = NILAIPERTAMA / NILAIKEDUA);     
        }
        else {
            printf("Input anda salah, silahkan coba lagi\n");

        }
    }
}



